### PR TITLE
changed oem file to python3

### PR DESCRIPTION
**Pull request by ashwin9390**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,170 +1,170 @@
...
+clean:
+	rm -f *.pyc
+	rm -f */*.pyc
+	rm -rf __pycache__
+
+depend:
+
+.PHONY: test
+test:
+	python -m unittest discover
```
Comment (AaronRobson) [inline Makefile:157]: Please update the `make test` target to:
```
.PHONY: test
test:
	$(PYTHON) -m unittest discover
```

With this at the top of the file:
```
ifneq (, $(shell which python3))
    PYTHON := python3
else
    PYTHON := python
endif
```

Then it should work on linux systems where `python3` is about or Windows where it is typically installed as `python`.

File: .travis.yml [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-  sudo mkdir -p /etc/sysconfig
-  sudo bash -c 'echo -e "NETWORKING=yes\nNETWORKING_IPV6=no\nIPV6_AUTOCONF=no\nHOSTNAME=localhost.localdomain" > /etc/sysconfig/network'
-
-script:
-  flake8 --select W291,W292 .
-  nosetests
+language: python
+
+python:
+  - 2.7
```
Comment (AaronRobson) [inline .travis.yml:4]: Please include at least the target version of python3 and consider including the latest (currently 3.11).

File: Makefile (hunk 1 above)
Comment (andyhhp) [reply Makefile:157]:
```make
# Best effort attempt to find a python interpreter, defaulting to Python 3 if
# available.  Fall back to just `python` if `which` is nowhere to be found.
PYTHON := $(word 1,$(shell which python3 python python2 2>/dev/null) python)
```

File: README.md
Comment (AaronRobson) [inline README.md:1]: Please don't change the line endings, it makes it a much noisier diff.
I recommend sticking with LF and enforcing it in the repo with a `.gitattributes` like:
```
*.md     text eol=lf
*.py     text eol=lf
*.txt     text eol=lf
*.yml     text eol=lf
```

File: XSConsoleLangworked.py
Comment (AaronRobson) [inline XSConsoleLangworked.py:1]: I think that this might have been commited by accident.

File: XSConsoleLang.py [hunk 3]
```diff
@@ -82,15 +82,15 @@ def ToString(cls, inLabel):
         elif isinstance(inLabel, Exception):
             exn_strings = []
             for arg in inLabel.args:
-                if isinstance(arg, unicode):
-                    exn_strings.append(arg.encode('utf-8'))
-                else:
+                #if isinstance(arg, unicode):
+                   # exn_strings.append(arg.encode('utf-8'))
+                #else:
```
Comment (AaronRobson) [inline XSConsoleLang.py:87]: You shouldn't have this commented out code.

File: XSConsoleTerm.py [hunk 4]
```diff
@@ -139,16 +139,16 @@ def Enter(self):
                             os.system(self.layout.ExitCommand())
                             Data.Inst().Update() # Pick up changes caused by the subshell command
 
-            except KeyboardInterrupt, e: # Catch Ctrl-C
+            except KeyboardInterrupt as e: # Catch Ctrl-C
                 XSLog('Resetting due to Ctrl-C')
                 Data.Reset()
-                sys.stderr.write("\033[H\033[J"+Lang("Resetting...")) # Clear screen and print banner
+                sys.stderr.write("\033[H\033[J"+Lang("Resetting...")) # Clear screen and print banner #Ashwin removed decode
```
Comment (AaronRobson) [inline XSConsoleTerm.py:145]: There wasn't a decode there anyway.

**Comment by AaronRobson** [comment]
I wouldn't feel comfortable merging this without significant additional unit testing.

**Comment by AaronRobson** [comment]
There is also far too much going on in this one PR.

**Pull request closed without merging**